app.py

```diff
@@ -31,8 +32,9 @@
     else:
         context = (config['certFile'], config['keyFile'])
 
-    app = connexion.App(__name__, specification_dir='./swagger/')
+    app = connexion.FlaskApp(__name__, specification_dir='./swagger/')
     app.add_api('swagger.yaml', arguments={'title': 'UCS Service'})
+    ThreadingTimer.ThreadingTimer(app)
```

Comment:
If you have no methods/attributed that will be used, a function looks enough for me.